README.md

```diff
@@ -26,17 +26,19 @@ near <command>
 #### For account:
 ```bash
   near login                                       # logging in through NEAR protocol wallet
-  near create_account <accountId>                  # create a developer account with --masterAccount(required), publicKey and initialBalance
```

Comment:
we really need a PR review checklist so that we don't get out of date next time (I am not expecting this to be done as part of this PR, just food for thought)